interface/src/app/mlmodel/resource.js

```diff
@@ -0,0 +1,9 @@
+angular.module('adage.mlmodel.resource', ['ngResource'])
+
+.factory('MLModel', ['$resource', function($resource) {
+  return $resource(
+    '/api/v0/mlmodel/'
+  );
+}])
```

Comment:
I'm fine with that. As I said, it's a weak recommendation and I can see it working both ways. The only thing I'll add here is that I think the resource is just as loosely-coupled whether it lives in this file or in the other one. The level of coupling has more to do with its abstraction as a separate component than what file it lives in.